### ISSUE TITLE
@nrwl/workspace:run-commands breaks with 12.6.5

### ISSUE THREAD

**Issue by smasala** [nrwl/nx#6661]
## Current Behavior

When running a step which is configured with `run-commands` the new version passes `--verbose=false` by default to all commands. This wasn't the case with v12.3.6

Compodoc fails now because it does not recognise this argument as it's passed to the command itself.

## Expected Behavior

Not to pass unwanted arguments to each command

## Steps to Reproduce

```
"build-storybook": {
          "builder": "@nrwl/workspace:run-commands",
          "options": {
            "commands": ["yarn run compodoc", "... further commands ..."],
            "parallel": false
          }
        }
```

**Workaround:**
To force the passing of an insignificant argument so that the `verbose` argument isn't pass to it - see the nx `parseArgs` in `run-commands.impl`.
```
"build-storybook": {
          "builder": "@nrwl/workspace:run-commands",
          "options": {
            "commands": ["yarn run compodoc --{args.silent}", "... further commands ..."],
            "args": "--silent=silent",
            "parallel": false
          }
        }
```

### Environment

  Node : 14.16.0
  OS   : darwin x64
  yarn : 1.22.10
  
  nx : Not Found
  @nrwl/angular : 12.6.5
  @nrwl/cli : 12.6.5
  @nrwl/cypress : 12.6.5
  @nrwl/devkit : 12.6.5
  @nrwl/eslint-plugin-nx : 12.6.5
  @nrwl/express : Not Found
  @nrwl/jest : 12.6.5
  @nrwl/linter : 12.6.5
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 12.6.5
  @nrwl/nx-cloud : 12.3.6
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 12.6.5
  @nrwl/web : 12.6.5
  @nrwl/workspace : 12.6.5
  @nrwl/storybook : 12.6.5
  @nrwl/gatsby : Not Found
  typescript : 4.3.5

**Comment by AgentEnder** [comment]
This has been fixed in the latest master, and should be fixed in 12.6.6. The issue was being tracked here: https://github.com/nrwl/nx/issues/6606

**Comment by smasala** [comment]
Thanks @AgentEnder
